src/providers/firestore.ts

```diff
@@ -214,3 +216,55 @@ export function objectToValueProto(data: object) {
 
   return mapValues(data, encodeHelper);
 }
+
+const FIRESTORE_ADDRESS_ENVS = [
+  'FIRESTORE_EMULATOR_HOST',
+  'FIREBASE_FIRESTORE_EMULATOR_ADDRESS',
+];
+
+const FIRESTORE_ADDRESS = FIRESTORE_ADDRESS_ENVS.reduce(
+  (addr, name) => process.env[name] || addr,
+  'localhost:8080'
+);
+const FIRESTORE_PORT = FIRESTORE_ADDRESS.split(':')[1];
+
+export type ClearFirestoreDataOptions = {
```

Comment:
Since this type just wraps projectId, I'd prefer to get rid of it and pass in projectId to clearFirestoreData.

Reply:
In order to avoid confusion. I think it would be good to keep it consistent with `@firebase/testing`.

Perhaps whe could allow both `clearFirestoreData({ projectId: "project-id" });` and `clearFirestoreData("project-id");`